EmailRequestAll/EmailRequestAll.py

```
@@ -0,0 +1,8 @@
+import requests 
+class EmailRequestAll:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)